app/helpers/home_helper.rb

```diff
@@ -37,6 +37,9 @@ def pretty_labels_for(paper)
           concat content_tag(:span, label, style: "padding: 3px; margin-right: 3px; border-radius: 2px; background-color: ##{colour}; color: white;")
         elsif label == "recommend-accept"
           concat content_tag(:span, label, style: "padding: 3px; margin-right: 3px; border-radius: 2px; background-color: ##{colour}; color: #000000;")
+        elsif label == "query-scope"
```

Comment:
This change means that papers labeled with `query-scope` on the `joss-reviews` repository will show the label in the dashboard.